### PR TITLE
[Feat] Ops cli with tree backfiller

### PR DESCRIPTION
**Pull request by kespinola**
resolves https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure/issues/150

## Goal
Implement a rapid, efficient, and reliable method for backfilling any missing updates to bubblegum trees.

## Changes
- Introduce an ops crate that consolidates various operational tools utilized by DAS operators.
- Implement a bubblegum backfill command. This command detects absent trees and their sequences within the index by referencing cl_audits_v2. It then requests missing transactions for a tree using getSignaturesForAddress to requeue any missed transactions. For usage details and methodology, refer to the bubblegum module's [README](https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure/blob/78479394b8465d9353515513bdc9a986b6079316/ops/src/bubblegum/README.md).

## Testing
At Triton, the backfill command successfully populated a new DAS installation with all bubblegum transactions in ~30 hours. The duration of indexing is contingent on the number of worker processes and the RPC response time, especially when retrieving older transactions from the archive.

The command has also been applied incrementally over the past month to backfill only missing transactions, taking between 2 to 5 minutes depending on the number and size of missing sequence gaps.

## Discussion
1. I welcome any alternative suggestions for organizing DAS-related tools. With community approval, I plan to merge the various tool crates into this ops package.

### CODE REVIEW

**Review by pmantica11 — APPROVED** [review]
This looks pretty high quality overall. The documentation is thorough and the code is pretty clear.  It also seems pretty safe since it doesn't affect the core components: the API and the ingester. I also like the `core` module. It is going to be useful to stop code duplication.

**Review by pmantica11 — APPROVED** [review]

**Comment by kespinola** [comment]
can remove backfill_items from upstream to resolve this issue once released and adopted by other providers.

https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure/issues/171

**Review by danenbm — APPROVED** [review]
LGTM, thank you this is awesome!  Left one minor question but approved!

File: ops/src/bubblegum/queue.rs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    pub async fn try_from_config(config: QueueArgs) -> anyhow::Result<Self, QueuePoolError> {
+        let size = usize::try_from(config.messenger_queue_connections)?;
+        let (tx, rx) = mpsc::channel(size);
+
+        for _ in 0..config.messenger_queue_connections {
+            let messenger_config: MessengerConfig = config.clone().into();
+            let mut messenger = plerkle_messenger::select_messenger(messenger_config).await?;
+            messenger.add_stream(TRANSACTION_BACKFILL_STREAM).await?;
+            messenger
+                .set_buffer_size(TRANSACTION_BACKFILL_STREAM, 10000000000000000)
```
[the remark this answers is not in the document]
Comment (kespinola) [reply ops/src/bubblegum/queue.rs]: ty. adjusted to the same size set in the current backfiller struct.

**Pull request merged**